packages/paste-core/components/pagination/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,247 @@
+import * as React from 'react';
+import {render, fireEvent} from '@testing-library/react';
+// @ts-ignore typescript doesn't like js imports
+import axe from '../../../../../.jest/axe-helper';
+import {
+  Pagination,
+  PaginationItems,
+  PaginationArrow,
+  PaginationLabel,
+  PaginationEllipsis,
+  PaginationNumber,
+  PaginationNumbers,
+} from '../src';
+
+describe('Pagination', () => {
+  const MockPagingationItems: React.FC = () => {
+    return (
+      <PaginationItems>
+        <PaginationArrow />
+        <PaginationLabel>Page 1 of 10</PaginationLabel>
+        <PaginationArrow direction="right" />
+      </PaginationItems>
+    );
+  };
+
+  const MockPagingationNumbers: React.FC = () => {
+    return (
+      <PaginationNumbers pageLabel="Page 1 of 3">
+        <PaginationNumber aria-label="Go to page 1">1</PaginationNumber>
+        <PaginationNumber aria-label="Go to page 2">2</PaginationNumber>
+        <PaginationNumber aria-label="Go to page 3">3</PaginationNumber>
+      </PaginationNumbers>
+    );
+  };
+
+  const MockArrowPaginationExample: React.FC = () => {
+    return (
+      <Pagination aria-label="pagination navigation">
+        <PaginationItems>
+          <PaginationArrow />
+          <PaginationLabel>Page 1 of 10</PaginationLabel>
+          <PaginationArrow direction="right" />
+        </PaginationItems>
+      </Pagination>
+    );
+  };
+
+  const MockNumberPaginationExample: React.FC = () => {
+    return (
+      <Pagination aria-label="pagination navigation">
+        <PaginationItems>
+          <PaginationArrow>Previous</PaginationArrow>
+          <PaginationNumbers pageLabel="Page 1 of 10">
+            <PaginationNumber aria-label="Go to page 1">1</PaginationNumber>
+            <PaginationEllipsis aria-label="Collapsed previous pages" />
+            <PaginationNumber aria-label="Go to page 4">4</PaginationNumber>
+            <PaginationNumber aria-label="Go to page 5" aria-current>
+              5
+            </PaginationNumber>
+            <PaginationNumber aria-label="Go to page 6">6</PaginationNumber>
+            <PaginationEllipsis aria-label="Collapsed next pages" />
+            <PaginationNumber aria-label="Go to page 10">10</PaginationNumber>
+          </PaginationNumbers>
+          <PaginationArrow direction="right">Next</PaginationArrow>
+        </PaginationItems>
+      </Pagination>
+    );
+  };
```

Comment:
Do you reckon it's possible to use some of the stories as Mock components, instead of creating these?

Reply:
Yup. I refactored the stories so they can be used in tests and elsewhere.